lib/src/kvs/ds.rs

```diff
@@ -283,6 +343,18 @@ impl Datastore {
 					info!("Connecting to kvs store at {}", path);
 					let s = s.trim_start_matches("fdb://");
 					let s = s.trim_start_matches("fdb:");
+
+					let s = match s.starts_with('~') {
+						true => s.replacen(
+							'~',
+							dirs::home_dir().unwrap().as_os_str().to_str().unwrap(),
+							1,
+						),
+						false => s.to_string(),
+					};
+
+					let s = s.as_str();
+
```

Comment:
`~` in paths are not relevant for FoundationDB.

Reply:
Removing the change.